flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/source/ScanContext.java

```diff
@@ -192,12 +194,12 @@ ScanContext copyWithAppendsBetween(long newStartSnapshotId, long newEndSnapshotI
         .project(schema)
         .filters(filters)
         .limit(limit)
-        .exposeLocality(exposeLocality)
```

Comment:
this reordering is to match the order where variables are defined. 

`copyWithSnapshotId(long newSnapshotId)` below also has the matching order